tests/parametric/conftest.py

```diff
@@ -201,13 +201,11 @@ def dotnet_library_factory():
         container_img=f"""
 FROM mcr.microsoft.com/dotnet/sdk:8.0
 
-RUN apt-get update && apt-get install -y dos2unix
-RUN apt-get update && apt-get install -y curl
+RUN apt-get update && apt-get install -y curl binutils
```

Comment:
[Note to reviewers]
- removed `dos2unix`, not needed anymore
- add `binutils`, required by `install_ddtrace.sh`
- combined the two packages into one line ([Docker best practice](https://docs.docker.com/build/building/best-practices/#apt-get))